core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -625,8 +632,7 @@ private void configureDosFilter(ServletContextHandler context) {
         "maxIdleTrackerMs", String.valueOf(config.getDosFilterMaxIdleTrackerMs().toMillis()));
     filterHolder.setInitParameter(
         "insertHeaders", String.valueOf(config.getDosFilterInsertHeaders()));
-    filterHolder.setInitParameter(
-        "trackSessions", String.valueOf(config.getDosFilterTrackSessions()));
+    filterHolder.setInitParameter("trackSessions", "false");
```

Comment:
This actually changes the value we've been using for this parameter which also changes the codepath we'll be using most of the time, even though we don't have a configured session manager. I don't think it will have an effect, but it's probably a good idea for you to check the `DoSFilter` code too for that (if you haven't already).

Reply:
As I said in the PR description, `trackSessions` has no effect, since we don't use sessions. Might as well disable it explicitly.